src/index.tsx

```diff
@@ -1,37 +1,103 @@
-import React, { useContext, useEffect, useRef, useState } from 'react';
+import { DataProxy } from 'apollo-cache';
+import ApolloClient, {
+  ApolloCurrentResult,
+  ApolloQueryResult,
+  FetchMoreOptions,
+  FetchMoreQueryOptions,
+  FetchPolicy,
+  MutationOptions,
+  ObservableQuery,
+  OperationVariables,
+  QueryOptions,
+  UpdateQueryOptions,
+} from 'apollo-client';
+import { FetchResult } from 'apollo-link';
+import { DocumentNode } from 'graphql';
+import React, {
+  ReactElement,
+  ReactNode,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
 import isEqual from 'react-fast-compare';
-
+import { Omit, assertApolloClient } from './Utils';
 import objToKey from './objToKey';
 import {
   getCachedObservableQuery,
   invalidateCachedObservableQuery,
 } from './queryCache';
 
-const ApolloContext = React.createContext();
+const ApolloContext = React.createContext<null | ApolloClient<any>>(null);
+
+export interface ApolloProviderProps {
```

Comment:
Shouldn't it be parametrized like in the old definitions: <https://github.com/trojanowski/react-apollo-hooks/blob/e1a9b6f77d2f9fb8c34ad89ee0ec5e3720e1b16d/src/index.d.ts#L25>?